collections/_article/dev-snapshot-godot-4-4-dev-7.md

```diff
@@ -0,0 +1,126 @@
+
+-----
+
+*The cover illustration is from* [**Pest Apocalypse**](https://store.steampowered.com/app/2506810/Pest_Apocalypse/), *a hyper-realistic post apocalyptic physics based pizza delivery action rogue-lite! It is developed by [Kikimora Games](https://www.kikimora.games/). You can purchase the game [on Steam](https://store.steampowered.com/app/2506810/Pest_Apocalypse/), and follow the developers [on BlueSky](https://bsky.app/profile/kikimora.games) and [Discord](https://discord.com/invite/sSmGbJSa4E).*
+
+## Highlights
+
+In case you missed them, see the [4.4 dev 1](/article/dev-snapshot-godot-4-4-dev-1/), [4.4 dev 2](/article/dev-snapshot-godot-4-4-dev-2/),
+[4.4 dev 3](/article/dev-snapshot-godot-4-4-dev-3/), [4.4 dev 4](/article/dev-snapshot-godot-4-4-dev-4/), [4.4 dev 5](/article/dev-snapshot-godot-4-4-dev-5/), and [4.4 dev 6](/article/dev-snapshot-godot-4-4-dev-6/) release notes for an overview of
+some key features which were already in those snapshots, and are therefore still available for testing in dev 7.
+
+Here are highlights of a few new features in dev 7 that you might find particularly exciting!
+
+### Jolt Physics Module
+
+Ever since its inception in late 2022, [godot-jolt](https://github.com/godot-jolt/godot-jolt) has become the de-facto 3D physics engine for many of our developers. Much of the history behind why this was the case is documented in [this proposal](https://github.com/godotengine/godot-proposals/issues/7308) by [Adam Scott](https://github.com/adamscott), but the main takeaway was a strong push for this tool to be recognized as an official addon. In doing so, users would be able to find this amazing resource in a way that was promoted by the engine itself; an exciting prospect!
+
+The Godot Jolt maintainers, [Mikael Hermansson](https://github.com/mihe) and [Jorrit Rouwe](https://github.com/jrouwe), took this idea one step further: integrating the tool as part of the engine *directly*. There was already a symbiosis between their team and the godot engine, with many features being added to Godot *and* Jolt to accomodate both, but the integration of an official module was no small feat; their pull request ([GH-99895](https://github.com/godotengine/godot/pull/99895)) ended up adding over 500 files and 115 **thousand** lines of code! So while this was one of the most rigorously tested PRs relative to the amount of time it's been up, it'd be impossible for any team to account for *everything* this behemoth introduced, so we eagerly await your feedback (and bug reports) on one of the most highly-requested features of 4.x.
+
+**NOTE:** At time of writing, this does **not** replace Godot Physics as the default 3D physics engine. This is because Jolt Physics have yet to reach full feature-parity with Godot Physics. If your interests/use-case are supported, the tool can be enabled by changing the `physics/3d/physics_engine` project setting to `Jolt Physics`.
```

Comment:
```suggestion
**NOTE:** At time of writing, this does **not** replace Godot Physics as the default 3D physics engine. This is because Jolt Physics has yet to reach full feature-parity with Godot Physics. If your interests/use-case are supported, the tool can be enabled by changing the `physics/3d/physics_engine` project setting to `Jolt Physics`.
```
At least, I *think* it should be "has".